components/camel-rocketmq/src/main/java/org/apache/camel/component/rocketmq/AclUtils.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.camel.component.rocketmq;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.rocketmq.acl.common.AclClientRPCHook;
+import org.apache.rocketmq.acl.common.SessionCredentials;
+import org.apache.rocketmq.remoting.RPCHook;
+
+public final class AclUtils {
+
+    private AclUtils() {
+    }
+
+    public static RPCHook getAclRPCHook(String accessKey, String secretKey) {
+        if (StringUtils.isNotBlank(accessKey) && StringUtils.isNotBlank(secretKey)) {
```

Comment:
Maybe replace this `StringUtils.isNotBlank` with Camel's `StringHelper.notBlank(String, String, String)` and prevent the NPE that could happen on ```RocketMQConsumer.startConsumer```?

Reply:
Actually I also want to avoid using null. But the constructors of RocketMQ classes accept null as default value. Passing null into RocketMQ's classes will not cause NPE.

Reply:
Great, in this case it should be fine. Thanks!